catalyst-gateway/bin/src/service/common/types/cardano/transaction_id.rs

```diff
@@ -0,0 +1,84 @@
+//! Transaction ID.
+
+use std::sync::LazyLock;
+
+use cardano_blockchain_types::TransactionId;
+use catalyst_types::hashes::BLAKE_2B256_SIZE;
+use poem_openapi::{
+    registry::{MetaSchema, MetaSchemaRef},
+    types::{Example, ParseError, ParseFromJSON, ParseFromParameter, ParseResult, ToJSON, Type},
+};
+use serde_json::Value;
+
+use crate::service::{common::types::string_types::impl_string_types, utilities::as_hex_string};
+
+/// Title.
+const TITLE: &str = "Transaction Id";
```

Comment:
Just to avoid confusion


```suggestion
const TITLE: &str = "Transaction Id or Transaction hash";
```